From 346646c7717a473d98812b0958a05fedcf484ef6 Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Mon, 13 Jul 2020 02:42:28 +0200
Subject: [PATCH 208/346] arm64: dts: sun50i-a64-pinephone: Enable HDMI audio
 on PinePhone

Now that HDMI audio patches are available, enable support for
PinePhone.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
index 21831ab90279..2a4554829f58 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
@@ -824,6 +824,14 @@ &tcon1 {
 
 ///////////// HDMI ///////////////
 
+&hdmi_sound {
+	status = "okay";
+};
+
+&i2s2 {
+	status = "okay";
+};
+
 &hdmi {
 	hvcc-supply = <&reg_dldo1>;
 	status = "okay";
-- 
2.25.1

